lang/es_ES.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -4590,8 +4590,8 @@ msgid "LDAP User Import debug"
 msgstr "Depuración de las importaciones de usuarios LDAP"
 
 #: centreon-web/www/include/Administration/parameters/debug/form.php:62
-msgid "Centcore Engine debug"
-msgstr "Depurando el motor de Centcore"
+msgid "Centreon Gorgone debug"
+msgstr "Depurando de Centreon Gorgone "
```

Comment:
are you sure for the "de" ?

Reply:
no it's the 'de' of:  motor **de** Centcore